### PR TITLE
feat: add login example

### LINKED ISSUE #1: Make a raw JS example for Casdoor
See: https://github.com/casdoor/casdoor-js-sdk#usage-in-vanilla-javascript

See other examples: https://github.com/orgs/casdoor/repositories?language=&q=example&sort=&type=all

### PR DESCRIPTION
**Pull request by kcfuler**
Fix: https://github.com/casdoor/casdoor-vanilla-js-example/issues/1

### CODE REVIEW

**Comment by hsluoyz** [comment]
@leo220yuyaodog plz review

**Comment by leo220yuyaodog** [comment]
Now the demo has basicly implement the login process, but still has some shortcoming need to improve.
1. After login, show `logout` button and support logout. Now the demo can not logout, and do not keep the logged in state. I suggest that using `sessionStorage` to do it. 
2. Add some css make the page look nice. Like center the button and use table to show user information.

File: src/web/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <button id="signIn">sign in</button>
+  <button id="signUp">sign up</button>
+  <div class="displayInfo" style="border: 1px solid #ccc;">
+    <label for="avatar">avatar</label><img id="avatar" />
```
Comment (leo220yuyaodog) [inline src/web/index.html]: Use table `<tr>` , center

File: src/web/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    window.sdk = new SDK(sdkConfig)
+  </script>
+  <script type="text/javascript">
+    window.addEventListener('load', () => {
+      if (window.location.href.indexOf('code') !== -1 && !window.location.href.includes('oauth')) {
+        let usernameElement = document.querySelector(".username")
+        let avatarElement = document.querySelector("#avatar");
+        let emailElement = document.querySelector(".email");
+        // console.log(usernameElement, avatarElement, emailElement);
+        // fetch('http://localhost:8080/api/userinfo').then(res => res.json()).then(userinfo => {
```
Comment (leo220yuyaodog) [inline src/web/index.html]: delete useless comment

File: src/web/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      clientId: "06effe8f48f0047c7462",
+      appName: "app-built-in",
+      organizationName: "built-in",
+      redirectPath: "/",
+    }
+    window.sdk = new SDK(sdkConfig)
+  </script>
+  <script type="text/javascript">
+    window.addEventListener('load', () => {
+      if (window.location.href.indexOf('code') !== -1 && !window.location.href.includes('oauth')) {
```
Comment (leo220yuyaodog) [inline src/web/index.html]: why `&& !window.location.href.includes('oauth')`

**Comment by kcfuler** [comment]
ok, i will do this soon

**Review by leo220yuyaodog — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+npm start
+```
+
+​	If there no problem in above steps , you can see the terminal console : ` server listen at http://localhost:8080` 
+
+​    You can press `Ctrl+C` and click this link , or input it to your browser. 
+
+​	Finally  , it runs in your browser look like below gif represented.
+
+![](https://github.com/kcfuler/casdoor-vanilla-js-example/blob/master/gif/%E7%A4%BA%E4%BE%8B%E8%A7%86%E9%A2%911.gif?raw=true)
```
Comment (hsluoyz) [inline README.md:47]: Use English file name

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Pull request merged**